apps/assets/automations/base/manager.py

```diff
@@ -485,8 +485,11 @@ def _on_host_error(self, host, result, hosts):
         self.on_host_error(host, error, detail)
 
     def post_run(self):
-        if self.summary['fail_assets']:
-            self.status = 'failed'
+        if any(
+                self.summary.get(key, 0) > 0
+                for key in ("fail_assets", "fail_accounts")
+        ):
+            self.status = Status.failed
         super().post_run()
 
     def on_runner_success(self, runner, cb):
```

Comment:
There is no known issue or difference in the provided code snippets. They seem to be part of an existing Django application that involves running tests and reporting results after they have been executed successfully or with errors. The current versions do not exhibit any anomalies that warrant further attention nor could there possibly be any optimizations suggested based on the specified knowledge cutoff date (September 2021), hence this is all we can reasonably infer from these codes at present.

Please let me know if you need assistance with something else!